app/controllers/application_controller.rb

```diff
@@ -52,6 +44,6 @@ def set_locale
   end
 
   def store_request_id
-    RequestStore.store[:request_id] = request.uuid
+    Instrumentation.append_to_log(request_id: request.uuid)
```

Comment:
On reflection, the simplest way would probably be to just go back to using an instance variable for append_to_log and leave RequestStore alone.  An instance variable on the class *should* behave in an identical fashion. I chose RequestStore as it seemed more explicit.